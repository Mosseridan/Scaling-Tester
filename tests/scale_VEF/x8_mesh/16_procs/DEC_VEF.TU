Statistiques de post resolution

Temps total                       4.36773

