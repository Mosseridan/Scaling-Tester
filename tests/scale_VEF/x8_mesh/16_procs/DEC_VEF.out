-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF
 
Reading data file 

14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
15 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

     29376     113952     113952 [20] [     2203:   29376 ]
     15413      79976      86026 [20] [     2203:   29376 ]
      8259      53920      67976 [20] [     2203:   29376 ]
      4423      34834      54080 [20] [     2203:   29376 ]
      2365      21270      44254 [20] [     2203:   29376 ]
      1252      12224      36026 [20] [     2203:   29376 ]
       658       6632      28712 [20] [     2203:   29376 ]
       345       3404      22608 [20] [     2203:   29376 ]
       180       1736      18064 [20] [     2203:   29376 ]
        94        866      14170 [20] [     2203:   29376 ]
        50        416      11474 [20] [     2203:   29376 ]
        27        188       9206 [20] [     2203:   29376 ]
        18        104       8292 [20] [     2203:   29376 ]
     14689      56740      56740 [20] [     1101:   14689 ]
      7707      39720      42776 [20] [     1101:   14689 ]
      4130      26726      33730 [20] [     1101:   14689 ]
      2196      17180      26712 [20] [     1101:   14689 ]
      1169      10418      21832 [20] [     1101:   14689 ]
       614       5864      17640 [20] [     1101:   14689 ]
       325       3130      13990 [20] [     1101:   14689 ]
       172       1580      11062 [20] [     1101:   14689 ]
        93        824       8686 [20] [     1101:   14689 ]
        49        424       6772 [20] [     1101:   14689 ]
        26        208       5464 [20] [     1101:   14689 ]
        18        140       4900 [20] [     1101:   14689 ]
      7344      28130      28130 [20] [      550:    7344 ]
      3861      19644      21164 [20] [      550:    7344 ]
      2071      13062      16610 [20] [      550:    7344 ]
      1101       8372      13020 [20] [      550:    7344 ]
       590       5082      10584 [20] [      550:    7344 ]
       312       2814       8582 [20] [      550:    7344 ]
       161       1474       6750 [20] [      550:    7344 ]
        86        758       5250 [20] [      550:    7344 ]
        46        380       4052 [20] [      550:    7344 ]
        25        186       3102 [20] [      550:    7344 ]
        19        138       2874 [20] [      550:    7344 ]
      3672      13882      13882 [20] [      275:    3672 ]
      1929       9620      10396 [20] [      275:    3672 ]
      1036       6366       8126 [20] [      275:    3672 ]
       554       4006       6360 [20] [      275:    3672 ]
       300       2384       5134 [20] [      275:    3672 ]
       159       1280       4080 [20] [      275:    3672 ]
        85        658       3258 [20] [      275:    3672 ]
        45        318       2440 [20] [      275:    3672 ]
        25        156       1864 [20] [      275:    3672 ]
        19        118       1786 [20] [      275:    3672 ]
      3672      13910      13910 [20] [      275:    3672 ]
      1928       9664      10422 [20] [      275:    3672 ]
      1042       6386       8218 [20] [      275:    3672 ]
       557       4086       6414 [20] [      275:    3672 ]
       304       2414       5228 [20] [      275:    3672 ]
       166       1364       4160 [20] [      275:    3672 ]
        87        714       3206 [20] [      275:    3672 ]
        47        340       2424 [20] [      275:    3672 ]
        25        158       1866 [20] [      275:    3672 ]
        20        124       1760 [20] [      275:    3672 ]
      7345      28252      28252 [20] [      550:    7345 ]
      3835      19698      21232 [20] [      550:    7345 ]
      2061      13190      16750 [20] [      550:    7345 ]
      1094       8484      13262 [20] [      550:    7345 ]
       587       5180      10820 [20] [      550:    7345 ]
       317       2976       8816 [20] [      550:    7345 ]
       165       1574       6968 [20] [      550:    7345 ]
        86        778       5390 [20] [      550:    7345 ]
        45        392       4148 [20] [      550:    7345 ]
        24        190       3148 [20] [      550:    7345 ]
        19        138       2906 [20] [      550:    7345 ]
      3672      13964      13964 [20] [      275:    3672 ]
      1932       9758      10484 [20] [      275:    3672 ]
      1037       6454       8242 [20] [      275:    3672 ]
       555       4126       6498 [20] [      275:    3672 ]
       295       2478       5286 [20] [      275:    3672 ]
       156       1372       4208 [20] [      275:    3672 ]
        82        692       3266 [20] [      275:    3672 ]
        43        344       2508 [20] [      275:    3672 ]
        22        166       1892 [20] [      275:    3672 ]
        20        148       1844 [20] [      275:    3672 ]
      3673      13962      13962 [20] [      275:    3673 ]
      1926       9694      10468 [20] [      275:    3673 ]
      1034       6444       8238 [20] [      275:    3673 ]
       560       4124       6518 [20] [      275:    3673 ]
       298       2400       5246 [20] [      275:    3673 ]
       159       1344       4202 [20] [      275:    3673 ]
        83        664       3186 [20] [      275:    3673 ]
        44        336       2456 [20] [      275:    3673 ]
        23        152       1870 [20] [      275:    3673 ]
        19        126       1738 [20] [      275:    3673 ]
     14687      56808      56808 [20] [     1101:   14687 ]
      7727      39930      42888 [20] [     1101:   14687 ]
      4137      26734      33894 [20] [     1101:   14687 ]
      2207      17202      26898 [20] [     1101:   14687 ]
      1184      10498      21952 [20] [     1101:   14687 ]
       625       5988      17916 [20] [     1101:   14687 ]
       332       3234      14238 [20] [     1101:   14687 ]
       174       1660      11134 [20] [     1101:   14687 ]
        91        834       8826 [20] [     1101:   14687 ]
        47        410       6798 [20] [     1101:   14687 ]
        26        200       5388 [20] [     1101:   14687 ]
        19        138       5044 [20] [     1101:   14687 ]
      7344      28220      28220 [20] [      550:    7344 ]
      3856      19826      21244 [20] [      550:    7344 ]
      2069      13252      16810 [20] [      550:    7344 ]
      1106       8492      13358 [20] [      550:    7344 ]
       588       5116      10916 [20] [      550:    7344 ]
       315       2932       8810 [20] [      550:    7344 ]
       167       1582       6942 [20] [      550:    7344 ]
        88        818       5490 [20] [      550:    7344 ]
        46        398       4356 [20] [      550:    7344 ]
        24        196       3432 [20] [      550:    7344 ]
        19        148       3290 [20] [      550:    7344 ]
      3672      13980      13980 [20] [      275:    3672 ]
      1920       9726      10476 [20] [      275:    3672 ]
      1025       6450       8220 [20] [      275:    3672 ]
       548       4126       6510 [20] [      275:    3672 ]
       295       2438       5294 [20] [      275:    3672 ]
       158       1376       4258 [20] [      275:    3672 ]
        84        726       3376 [20] [      275:    3672 ]
        45        362       2600 [20] [      275:    3672 ]
        23        176       2060 [20] [      275:    3672 ]
        20        158       1996 [20] [      275:    3672 ]
      3672      13898      13898 [20] [      275:    3672 ]
      1943       9696      10440 [20] [      275:    3672 ]
      1047       6482       8162 [20] [      275:    3672 ]
       561       4116       6406 [20] [      275:    3672 ]
       305       2454       5176 [20] [      275:    3672 ]
       160       1346       4136 [20] [      275:    3672 ]
        86        714       3176 [20] [      275:    3672 ]
        45        354       2514 [20] [      275:    3672 ]
        23        164       1878 [20] [      275:    3672 ]
        21        146       1834 [20] [      275:    3672 ]
      7343      28252      28252 [20] [      550:    7343 ]
      3866      19776      21298 [20] [      550:    7343 ]
      2075      13238      16790 [20] [      550:    7343 ]
      1111       8510      13330 [20] [      550:    7343 ]
       595       5172      10818 [20] [      550:    7343 ]
       312       2910       8674 [20] [      550:    7343 ]
       164       1578       6840 [20] [      550:    7343 ]
        89        816       5280 [20] [      550:    7343 ]
        48        418       4112 [20] [      550:    7343 ]
        25        194       3120 [20] [      550:    7343 ]
        19        146       2966 [20] [      550:    7343 ]
      3671      13952      13952 [20] [      275:    3671 ]
      1924       9672      10458 [20] [      275:    3671 ]
      1032       6400       8176 [20] [      275:    3671 ]
       552       4124       6418 [20] [      275:    3671 ]
       297       2460       5246 [20] [      275:    3671 ]
       160       1364       4244 [20] [      275:    3671 ]
        86        728       3338 [20] [      275:    3671 ]
        45        366       2462 [20] [      275:    3671 ]
        24        184       1940 [20] [      275:    3671 ]
        19        134       1850 [20] [      275:    3671 ]
      3672      13978      13978 [20] [      275:    3672 ]
      1928       9762      10490 [20] [      275:    3672 ]
      1036       6464       8254 [20] [      275:    3672 ]
       552       4070       6514 [20] [      275:    3672 ]
       298       2410       5274 [20] [      275:    3672 ]
       160       1326       4216 [20] [      275:    3672 ]
        86        710       3298 [20] [      275:    3672 ]
        45        334       2518 [20] [      275:    3672 ]
        25        172       1926 [20] [      275:    3672 ]
        20        136       1790 [20] [      275:    3672 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.162
     Coarsening: 		   0.081
            Matching: 			   0.027
            Contract: 			   0.047
     Initial Partition: 	   0.004
     Uncoarsening: 		   0.066
          Refinement: 			   0.046
          Projection: 			   0.020
     Splitting: 		   0.011
********************************************************************

 gk_mcore statistics
           coresize:       353544         nmops:         2048  cmop:      0
        num_callocs:         1624   num_hallocs:            0
       size_callocs:      6823320  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       352528   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

     28224     109984     109984 [20] [     2116:   28224 ]
     14824      77408      83184 [20] [     2116:   28224 ]
      7908      51952      65728 [20] [     2116:   28224 ]
      4207      33636      52262 [20] [     2116:   28224 ]
      2248      20690      42870 [20] [     2116:   28224 ]
      1181      11892      35004 [20] [     2116:   28224 ]
       616       6434      28038 [20] [     2116:   28224 ]
       326       3380      22184 [20] [     2116:   28224 ]
       170       1750      17708 [20] [     2116:   28224 ]
        88        898      14302 [20] [     2116:   28224 ]
        46        446      11604 [20] [     2116:   28224 ]
        25        204       9186 [20] [     2116:   28224 ]
        20        162       8688 [20] [     2116:   28224 ]
     14112      54552      54552 [20] [     1058:   14112 ]
      7400      38288      41128 [20] [     1058:   14112 ]
      3961      25554      32500 [20] [     1058:   14112 ]
      2110      16376      25684 [20] [     1058:   14112 ]
      1129       9978      20898 [20] [     1058:   14112 ]
       594       5644      16996 [20] [     1058:   14112 ]
       314       3028      13432 [20] [     1058:   14112 ]
       165       1524      10548 [20] [     1058:   14112 ]
        86        790       8412 [20] [     1058:   14112 ]
        46        394       6586 [20] [     1058:   14112 ]
        25        190       5368 [20] [     1058:   14112 ]
        18        128       5062 [20] [     1058:   14112 ]
      7056      27092      27092 [20] [      529:    7056 ]
      3716      19004      20412 [20] [      529:    7056 ]
      1991      12718      16076 [20] [      529:    7056 ]
      1071       8170      12734 [20] [      529:    7056 ]
       574       4964      10364 [20] [      529:    7056 ]
       308       2816       8352 [20] [      529:    7056 ]
       165       1512       6456 [20] [      529:    7056 ]
        87        768       5060 [20] [      529:    7056 ]
        46        380       3816 [20] [      529:    7056 ]
        25        186       2994 [20] [      529:    7056 ]
        18        130       2744 [20] [      529:    7056 ]
      3528      13356      13356 [20] [      264:    3528 ]
      1855       9318      10010 [20] [      264:    3528 ]
       989       6190       7834 [20] [      264:    3528 ]
       533       3886       6222 [20] [      264:    3528 ]
       285       2356       4984 [20] [      264:    3528 ]
       152       1266       4022 [20] [      264:    3528 ]
        83        674       3126 [20] [      264:    3528 ]
        46        346       2368 [20] [      264:    3528 ]
        25        174       1800 [20] [      264:    3528 ]
        18        116       1630 [20] [      264:    3528 ]
      3528      13412      13412 [20] [      264:    3528 ]
      1857       9336      10070 [20] [      264:    3528 ]
       994       6174       7886 [20] [      264:    3528 ]
       535       3914       6178 [20] [      264:    3528 ]
       288       2318       4980 [20] [      264:    3528 ]
       156       1288       4030 [20] [      264:    3528 ]
        82        650       3090 [20] [      264:    3528 ]
        44        324       2342 [20] [      264:    3528 ]
        23        166       1766 [20] [      264:    3528 ]
        19        132       1674 [20] [      264:    3528 ]
      7056      27126      27126 [20] [      529:    7056 ]
      3708      19004      20430 [20] [      529:    7056 ]
      1990      12762      16160 [20] [      529:    7056 ]
      1063       8196      12780 [20] [      529:    7056 ]
       568       4922      10422 [20] [      529:    7056 ]
       302       2784       8430 [20] [      529:    7056 ]
       158       1480       6648 [20] [      529:    7056 ]
        83        760       5162 [20] [      529:    7056 ]
        42        366       4044 [20] [      529:    7056 ]
        22        162       3078 [20] [      529:    7056 ]
        20        140       3006 [20] [      529:    7056 ]
      3528      13416      13416 [20] [      264:    3528 ]
      1856       9332      10072 [20] [      264:    3528 ]
       989       6210       7888 [20] [      264:    3528 ]
       523       3916       6228 [20] [      264:    3528 ]
       280       2344       5034 [20] [      264:    3528 ]
       147       1276       3964 [20] [      264:    3528 ]
        80        678       3096 [20] [      264:    3528 ]
        43        346       2426 [20] [      264:    3528 ]
        22        168       1888 [20] [      264:    3528 ]
        19        144       1802 [20] [      264:    3528 ]
      3528      13386      13386 [20] [      264:    3528 ]
      1852       9272      10034 [20] [      264:    3528 ]
       994       6150       7822 [20] [      264:    3528 ]
       529       3900       6092 [20] [      264:    3528 ]
       283       2290       4908 [20] [      264:    3528 ]
       150       1274       3932 [20] [      264:    3528 ]
        81        666       3038 [20] [      264:    3528 ]
        43        332       2242 [20] [      264:    3528 ]
        23        144       1598 [20] [      264:    3528 ]
        20        128       1538 [20] [      264:    3528 ]
     14112      54638      54638 [20] [     1058:   14112 ]
      7414      38372      41242 [20] [     1058:   14112 ]
      3950      25536      32496 [20] [     1058:   14112 ]
      2108      16434      25748 [20] [     1058:   14112 ]
      1134      10022      21092 [20] [     1058:   14112 ]
       595       5674      17106 [20] [     1058:   14112 ]
       310       3082      13606 [20] [     1058:   14112 ]
       163       1614      10646 [20] [     1058:   14112 ]
        86        814       8422 [20] [     1058:   14112 ]
        44        394       6630 [20] [     1058:   14112 ]
        23        188       5160 [20] [     1058:   14112 ]
        20        162       5050 [20] [     1058:   14112 ]
      7056      27124      27124 [20] [      529:    7056 ]
      3704      18878      20420 [20] [      529:    7056 ]
      1968      12574      16016 [20] [      529:    7056 ]
      1048       8076      12646 [20] [      529:    7056 ]
       563       4886      10320 [20] [      529:    7056 ]
       299       2708       8338 [20] [      529:    7056 ]
       159       1444       6580 [20] [      529:    7056 ]
        82        742       5156 [20] [      529:    7056 ]
        42        366       3982 [20] [      529:    7056 ]
        22        172       3014 [20] [      529:    7056 ]
        20        156       2992 [20] [      529:    7056 ]
      3528      13416      13416 [20] [      264:    3528 ]
      1855       9312      10070 [20] [      264:    3528 ]
       991       6152       7890 [20] [      264:    3528 ]
       530       3878       6212 [20] [      264:    3528 ]
       284       2294       5040 [20] [      264:    3528 ]
       149       1248       4012 [20] [      264:    3528 ]
        80        658       3050 [20] [      264:    3528 ]
        42        316       2350 [20] [      264:    3528 ]
        22        158       1748 [20] [      264:    3528 ]
        19        132       1670 [20] [      264:    3528 ]
      3528      13392      13392 [20] [      264:    3528 ]
      1850       9284      10036 [20] [      264:    3528 ]
       993       6200       7862 [20] [      264:    3528 ]
       532       3946       6190 [20] [      264:    3528 ]
       289       2360       4980 [20] [      264:    3528 ]
       154       1340       4042 [20] [      264:    3528 ]
        83        710       3110 [20] [      264:    3528 ]
        45        350       2428 [20] [      264:    3528 ]
        25        172       1866 [20] [      264:    3528 ]
        19        130       1700 [20] [      264:    3528 ]
      7056      27066      27066 [20] [      529:    7056 ]
      3694      18898      20342 [20] [      529:    7056 ]
      1975      12588      15964 [20] [      529:    7056 ]
      1061       8066      12648 [20] [      529:    7056 ]
       570       4800      10264 [20] [      529:    7056 ]
       303       2714       8252 [20] [      529:    7056 ]
       160       1478       6526 [20] [      529:    7056 ]
        83        742       5000 [20] [      529:    7056 ]
        44        366       3864 [20] [      529:    7056 ]
        23        174       3028 [20] [      529:    7056 ]
        20        148       2918 [20] [      529:    7056 ]
      3528      13318      13318 [20] [      264:    3528 ]
      1855       9248       9972 [20] [      264:    3528 ]
       995       6068       7796 [20] [      264:    3528 ]
       535       3820       6124 [20] [      264:    3528 ]
       288       2254       4886 [20] [      264:    3528 ]
       154       1218       3910 [20] [      264:    3528 ]
        83        634       3032 [20] [      264:    3528 ]
        45        316       2280 [20] [      264:    3528 ]
        25        154       1674 [20] [      264:    3528 ]
        18        106       1484 [20] [      264:    3528 ]
      3528      13340      13340 [20] [      264:    3528 ]
      1855       9282       9994 [20] [      264:    3528 ]
       989       6084       7804 [20] [      264:    3528 ]
       527       3848       6076 [20] [      264:    3528 ]
       284       2316       4924 [20] [      264:    3528 ]
       149       1216       3940 [20] [      264:    3528 ]
        78        614       3000 [20] [      264:    3528 ]
        41        288       2182 [20] [      264:    3528 ]
        22        138       1636 [20] [      264:    3528 ]
        19        116       1576 [20] [      264:    3528 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.129
     Coarsening: 		   0.075
            Matching: 			   0.035
            Contract: 			   0.033
     Initial Partition: 	   0.004
     Uncoarsening: 		   0.040
          Refinement: 			   0.027
          Projection: 			   0.012
     Splitting: 		   0.007
********************************************************************

 gk_mcore statistics
           coresize:       339720         nmops:         2048  cmop:      0
        num_callocs:         1616   num_hallocs:            0
       size_callocs:      6560544  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       338704   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

19 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 4.36773 s
